src/core/MOM.F90

```diff
@@ -2382,9 +2380,28 @@ subroutine initialize_MOM(Time, Time_init, param_file, dirs, CS, restart_CSp, &
       CS%tv%S => S_in
     endif
 
-    call MOM_initialize_state(u_in, v_in, h_in, CS%tv, Time, G_in, GV, US, &
-        param_file, dirs, restart_CSp, CS%ALE_CSp, CS%tracer_Reg, &
-        sponge_in_CSp, ALE_sponge_in_CSp, OBC_in, Time_in)
+    if (use_ice_shelf) then
+      allocate(frac_shelf_in(G_in%isd:G_in%ied, G_in%jsd:G_in%jed))
+      frac_shelf_in(:,:) = 0.0
+      allocate(CS%frac_shelf_h(isd:ied, jsd:jed))
+      CS%frac_shelf_h(:,:) = 0.0
+      call ice_shelf_query(ice_shelf_CSp,G,CS%frac_shelf_h)
+      ! MOM_initialize_state is using the  unrotated metric
+      call rotate_array(CS%frac_shelf_h, -turns, frac_shelf_in)
+      ! TODO: Verify that pass_var is needed here?
+      call pass_var(frac_shelf_in, G_in%Domain)
```

Comment:
Index rotation is currently unsupported for more than one MPI rank, due to some FMS rank assignment issues, so you may as well remove this line.

If we do get it working one day, then the `pass_var` should not be necessary.